docs/events/eboard-meetings.rst

```diff
@@ -0,0 +1,49 @@
+###############
+Eboard Meetings
+###############
+
+This page documents RITlug executive board (eboard) meetings.
+Topics include room booking, preparation, and running of the meeting.
+
+
+*****************
+Meeting checklist
+*****************
+
+Pre-meeting
+===========
+
+The President will triage the `RITlug operations board <https://github.com/orgs/RITlug/projects/1>`__, looking for tasks that require attention.
+The label **review at meeting** should be assigned to tasks that need explicit review by eboard.
+
+Each topic to be discussed at an eboard meeting must have a corresponding issue.
+
+Meeting
+=======
+
+#. Designate a notetaker, typically the `secretary <../roles/secretary>`__, to record discussion points and action items for each task.
```

Comment:
Oops! I just noticed this doesn't use the Sphinx cross-document syntax. The correct syntax here is:

```rst
:doc:`secretary <../roles/secretary>`
```